pkgs/servers/nextcloud/notify_push.nix

```diff
@@ -3,10 +3,14 @@
   fetchFromGitHub,
   nixosTests,
   rustPlatform,
+  fetchNextcloudApp,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "notify_push";
+
+  # NOTE: make sure this is compatible with all Nextcloud versions
+  # in nixpkgs!
```

Comment:
Maybe we can link https://github.com/nextcloud/notify_push/blob/main/appinfo/info.xml#L30 ?

They seem to support a very wide range so hopefully this doesn't become a problem.